lldb/source/Plugins/Process/minidump/ProcessMinidump.h

```diff
@@ -109,7 +109,7 @@ class ProcessMinidump : public PostMortemProcess {
 private:
   lldb::DataBufferSP m_core_data;
   llvm::ArrayRef<minidump::Thread> m_thread_list;
-  const minidump::ExceptionStream *m_active_exception;
+  std::unordered_map<uint32_t, minidump::ExceptionStream> m_exceptions_by_tid;
```

Comment:
Store `const minidump::ExceptionStream&` instead.

Reply:
This will expose my noviceness in C++, but I struggled that you can't emplace a list of references, so instead I refactored this to return a vector of const pointers.